upgrade-2.x.sh

```diff
@@ -940,12 +1001,20 @@ target_version=$(echo "$target_version" | tr + _)
 if [ -z "$RESINOS_TAG" ]; then
     RESINOS_TAG=${target_version}-${SLUG}
 fi
-image="${RESINOS_REGISTRY}/${RESINOS_REPO}:${RESINOS_TAG}"
-log "Checking for manifest of ${image}"
-if [ "$(image_exists "$RESINOS_REGISTRY" "$RESINOS_REPO" "$RESINOS_TAG")" = "yes" ]; then
-    log "Manifest found, good to go..."
+log "Attempting host OS update using deltas"
+
+image=$(find_delta "${target_version/_/+}")
+if [ -z "${image}" ]; then
```

Comment:
I might suggest restructuring this a bit:

```
if [ "${image}" ] ; then
  log "Found delta located at ${image}"
else
  ...
```

This makes for a shorter (visual) distance to keep the context of the check, compared to having it after a failure check with its own nested check.

Reply:
Yep, that makes sense, I'll flip this logic. Good note!